### ISSUE TITLE
last.fm: Restrict playlist to specified tracks (user defined tags) ?

### ISSUE THREAD

**Issue by gitgetgot** [hymerman/foo_bestversion#30]
Would it be possible (and useful for others?) to restrict the generated playlists only to tracks, which contain a user defined tag as well?

Say I want the generated playlist only show matches from my library which files have the [%DISCO_GS_RELEASE_ID% Tag](https://www.wikidata.org/wiki/Property:P2206) set.

Or I want the playlist restricted to music with %DATE% Tags between 1989 - 1998

I like that to be optional and play around with different tags...

**Comment by hymerman** [comment]
That seems a bit special-purpose for this plugin... in my mind that seems like a job for another plugin, some kind of playlist filter, which probably already exists. I'll leave this open but probably won't be working on it any time soon, sorry!

**Comment by gitgetgot** [comment]
you're right, it could be done with additionally filtering your generated playlist.
There are plenty of filtering tools or components in foobar2000.

As I wrote this request, I did not think about it :-)
